modernomad/settings.py

```diff
@@ -211,4 +212,21 @@
     '--nologcapture'
 ]
 
+
+class DisableMigrations(object):
+    def __contains__(self, item):
+        return True
+
+    def __getitem__(self, item):
+        return None
+
```

Comment:
cool thanks. yeah, agree the package is overkill :). happy to go with this for now.